src/main/java/CarThread.java

```diff
@@ -0,0 +1,21 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarThread extends Thread {
+    List<Car> cars = new ArrayList<>();
+
+
+    public void setCars(List<Car> cars) { this.cars = cars; }
+    public void run() {
+        for (int i = 0; i < cars.size(); i++) {
+            cars.get(i).stopAndGo((int)(Math.random() * 10));
+            cars.get(i).printCar();
+        }
+        System.out.println();
+        try {
+            sleep(1000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
```

Comment:
```suggestion
            System.out.println(e.getMessage());
```
어디서 오류가 나오는지 모르기 때문에 메세지를 주는게 어떨까요? :)

Reply:
그렇게 수정하도록 하겠습니다!